src/global/meta.ts

```diff
@@ -8,16 +8,21 @@ import { ref, watch } from "vue";
  */
 
 /** variables */
-export const appTitle = ref(process.env.VUE_APP_TITLE);
+export const appTitle = ref([process.env.VUE_APP_TITLE]);
```

Comment:
I assume you changed this to an array because a string is immutable and thus not observable, but is that the actual reason? That'd be fine, but I wanted to check.

Reply:
Actually, I guess that's not quite it, since `ref()` can apparently take a string...I apologize, I don't know much about the Vue composition API. I'm curious to hear why you wrap this particular string in an array and not others, like the ones immediately below it.

Reply:
Yes ref can take any data type. The reason I changed `appTitle` to an array of strings instead of just a string is just for convenience:

```ts
const title = appTitle.value
    .concat([process.env.VUE_APP_TITLE])
    .filter((part) => part)
    .join(" | ");

  document.title = title || "";
```

This makes making multi-part titles, e.g. "Node Search | Marfan Syndrome | Monarch Initiative", a little more convenient imo, rather than having to do a bunch of if's for figuring out where to put the separators. It also makes it such that you only have to specify the separator (`|`) in one place, rather than anywhere where you set `appTitle`.

Reply:
Oh, duh, I didn't notice that code where you create the title from the array. Ok, that makes perfect sense, and I agree that an array is the right way to represent a title with multiple bits in it.

Reply:
Typescript will throw a warning if you try to set `appTitle` to just a string.

I'll make the comment above the declaration of `appTitle` more descriptive, e.g. "multi-part document title, as an array, that gets joined with | separator", so anyone who goes to set it will see that.

I'll add it to my next-next PR, since I am a few PRs ahead.